addon_service/tests/test_by_type/test_resource_reference.py

```diff
@@ -161,7 +161,7 @@ def test_get(self):
         with self.subTest("Confirm expected relationships"):
             self.assertEqual(
                 json.loads(_resp.rendered_content)["data"]["relationships"].keys(),
-                {"configured_storage_addons"},
+                {"configured_storage_addons", "configured_citation_addons"},
```

Comment:
@felliott Okay, maybe not. It's all over. Good luck, and may God have mercy on your soul.